StarCellBio/urls.py

```diff
@@ -34,10 +39,97 @@
                            {'document_root': 'html_app/images/'}),
                        url(r'^pdf/(?P<path>.*)$', 'django.views.static.serve', {'document_root': 'html_app/pdf/'}),
                        url(r'^js/(?P<path>.*)$', 'django.views.static.serve', {'document_root': 'html_app/js/'}),
-                       
+
 )
 # add authentication URL patterns
 urlpatterns += auth.urls.urlpatterns
+# urlpatterns += courseview.urlpatterns
+
+urlpatterns += patterns('',
```

Comment:
nit: all of these would be nicer formatted with:
```python
urlpatterns += patterns(
    '',
    url(r'^courses/$', 'backend.courseview.list', name='course_list'),
    ...
)
```
instead of that ton of white space